spharpy/classes/audio.py

```diff
@@ -241,6 +239,22 @@ def __init__(self, data, times, basis_type, normalization,
         TimeData.__init__(self, data=data, times=times, comment=comment,
                           is_complex=is_complex)
 
+    @classmethod
+    def from_spherical_harmonics_definition(
+            cls, data, times, sh_definition, domain='time',
+            fft_norm='none', comment="", is_complex=False):
+        r"""
+        Create a SphericalHarmonicsSignal class object from
+        SphericalHarmonicsDefinition object, data, and sampling
+        rate.
```

Comment:
Just noting this so it's not forgotten:

i) Please use the correct names, also see occurences below.
```suggestion
        Create a SphericalHarmonicSignal class object from
        SphericalHarmonicDefinition object, data, and sampling
        rate.
```
ii)  Classmethods should get parameter documentation